kafka_consumer/datadog_checks/kafka_consumer/client/kafka_client_factory.py

```diff
@@ -2,5 +2,5 @@
 from datadog_checks.kafka_consumer.client.kafka_python_client import KafkaPythonClient
 
 
-def make_client(check) -> KafkaClient:
```

Comment:
Since we are no longer passing in the entire check, we need to pass in the individual parts needed.